class.wp-cldr.php

```diff
@@ -49,19 +49,51 @@ public function set_locale( $locale ) {
 	*/
 	public function get_CLDR_locale( $wp_locale ) {
 
+		// this array captures the WordPress locales that are signficiantly different from CLDR locales
 		$wp2cldr =  array(
```

Comment:
> Are any of these WordPress.com-specific?

No. The WordPress.com locales are a subset of the broader set used across WordPress.org.

> Do you otherwise anticipate the need for plugin users to override these? This array might be a good place to do an apply_filter

I don't think other overriding will be necessary. Next PR allows for some fuzzy matching and we could include an option for a "strict" match to suppress that. But realistically the combined exceptions array and the generalization rules should cover all the currently used .org locales and almost all future ones.